jax/_src/numpy/fft.py

```diff
@@ -109,10 +109,76 @@ def _fft_core(func_name: str, fft_type: xla_client.FftType, a: ArrayLike,
   return transformed
 
 
-@implements(np.fft.fftn)
 def fftn(a: ArrayLike, s: Shape | None = None,
          axes: Sequence[int] | None = None,
          norm: str | None = None) -> Array:
+  r"""Compute the multidimensional discrete fourier transform.
```

Comment:
```suggestion
  r"""Compute the multidimensional discrete Fourier transform.
```

"Fourier" should be capitalized. But also, I wasn't immediately clear on what "multidimensional" meant here. Maybe we should clarify that it is the FFT of a multidimensional array along specified axes or something?

Reply:
"Compute the FFT of a multidimensional array along specified axes" sounds good to me.

Reply:
Hi @dfm, I think here "multidimensional discrete Fourier transform" seems to be the correct description. because even `jnp.fft.fft` computes the discrete Fourier transform of multidimensional array. 

`jnp.fft.fftn` computes FFT along multiple axes (hence called multidimensional FFT) whereas `jnp.fft.fftn` computes FFT along only one axis (hence called one-dimensional FFT). Could you please tell me whether my understanding is correct?

Reply:
I think you're right! I do find the terminology a bit confusing, but I think you're right that something like what you're suggesting would be more accurate. Would you be willing to open a new PR?

Reply:
I would be writing the docstrings for `jnp.fft.fft` and `jnp.fft.ifft`. I will include this change in that PR. Is that fine?

Reply:
Sounds good to me!